File: gradle/packages/cli-function/src/main/kotlin/com/app/cli/function/Banner.kt
package com.app.cli.function

import com.app.goodbye.world.printBanner

object Banner {
    @JvmStatic
    fun main(args: Array<String>) {
        println("===================================================")
        printBannerMessage()
        println("===================================================")
    }

    fun printBannerMessage() {
        println(printBanner())
    }
}

File: csharp/src/CLI.Function/Banner/Banner.cs
using Prog.Goodbye.World;

namespace Prog.CLI.Function;

public class Banner
{
    /// <summary> <h1><c>PrintBannerMessage</c></h1>
    /// <br/><br/>This method prints the banner message.<br/><br/>
    /// </summary>
    public static void PrintBannerMessage()
    {
        Console.WriteLine(GoodbyeWorld.PrintBanner());
    }
}
